addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -119,6 +119,27 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
         const googleLocalFontsProperty = weUtils.getCSSVariableValue('google-local-fonts', style);
         this.googleLocalFonts = googleLocalFontsProperty ?
             googleLocalFontsProperty.slice(1, -1).split(/\s*,\s*/g) : [];
+        this.headLinks = [];
+        const makeFontStyleElement = (fontURL) => {
+            const existingStyle = document.head.querySelectorAll(`link[href*="${fontURL}"]`);
+            if (!existingStyle.length) {
+                const link = document.createElement('link');
+                link.href = fontURL;
+                link.rel = 'stylesheet';
+                link.type = 'text/css';
+                this.headLinks.push(link);
+                document.head.prepend(link);
+            }
+        };
```

Comment:
There is an util in Odoo to do this if I am not mistaken: `loadCSS`. You might need some more code to be able to remove the links later on, but using `loadCSS` would be better.

Also your code is not robust enough in case you use two familipicker A and B:
- A start -> load css and save the link to remove
- B start -> load css... prevent it because already there, good
- A destroy -> destroy saved linked
- B does not work anymore, the links are gone (you cannot suppose B is destroyed before A, right? :thinking:)

Maybe the easiest thing for you to do is to not remove the loaded font CSS until the editor itself is destroyed: it make sense, no need of multiple CSS load in case 4 family pickers are used during the same edition.

Reply:
Indeed I should have seen that :/

I tried loading the CSS using `loadCSS`. It seems like the function is memoized, meaning that if I try to remove the links when the editor is destroyed, they won't ever be added again.

Should I make my own `<link>` elements or should I use `loadCSS` and never remove the links?

For reference I pushed a commit where I use `loadCSS` and adapted a little bit.